app/src/webtrigger/handle-jenkins-request.test.ts

```diff
@@ -2,11 +2,15 @@
  * This is an integration test of the whole code path from the webtrigger through the app
  * to the Jira API.
  */
+
+/* Updating error to warning as there are reasonable exceptions to the rule in this file */
```

Comment:
https://softwareteams.atlassian.net/browse/ARC-1433
Added ticket to address this rule - move JWT tokens to separate file and disable lint errors there instead